source/operationdefinition/operationdefinition-notes.xml

```diff
@@ -31,6 +31,64 @@ The derived operation can't change the nature of the operation itself, nor can
 it change add types to parameters or prohibit mandatory parameters.
 </p>
 
+<p>
+  When deriving from another operation, the following rules SHOULD be followed. Note that these rules
+  are asserted as SHOULD rather than SHALL only because they were introduced after the resource became 
+  normative. Systems that fail to follow these rules are likely to result in unsafe interoperability.
+</p>
+<ul>
+  <li>SHOULD NOT change the value of <code>affectsState</code> from the original OperationDefinition</li>
+  <li>SHOULD have the same <code>kind</code> as the original OperationDefinition</li>
+  <li>SHOULD have <code>experimental=false</code> if the original OperationDefinition has <code>experimental=true</code></li>

```

Comment:
Can you confirm this line? It doesn't immediately make sense, but I don't recall my thinking at the time.

Reply:
Typo from reconciling SearchParameter and OperationDefinition versions. Fixed in next push.